docs/tutorials/getting-started-eclipse.md

```diff
@@ -0,0 +1,71 @@
+---
+type: tutorial
+layout: tutorial
+title:  "Getting Started with Eclipse Luna"
+description: "This tutorial guides through installing Kotlin plugin for Eclipse and creating Hello World application."
```

Comment:
guides *us* through installing *the* Kotlin plugin for Eclipse *and* create *a* Hello World application"